src/main/content/antora_ui/src/js/06-javadocs.js

```diff
@@ -427,21 +443,57 @@ function updateTitle(currentPage) {
 }
 
 function addClickListeners() {
-  var iframes = $("#javadoc_container")
+  var main_frame = $("#javadoc_container");
+  var iframes = main_frame
```

Comment:
Revert this back to original code as the change is no longer needed
```suggestion
  var iframes = $("#javadoc_container");
```